docs/adr/0025-provide-testing-environments.md

```diff
@@ -0,0 +1,80 @@
+# *Provide a Testing Environment for Non-Engineers*
+
+**User Story:** *317* <!-- optional -->
+
+Because of the way job codes are configured, testing new features is
+becoming difficult. We've also run into situations where we had to
+track down users with specific job codes selected in order to test
+certain functionality. We need a better way!
+
+For the scope of this ADR, we are going to focus on the ability to test
+features, regardless of whether that testing happens before or after a
+merge to our main branch. Gating code merges is out of scope for now.
+
+## Considered Alternatives
+
+* *Use Launch Darkly to Selectively Downgrade Permissions*
+* *Do Testing Locally*
+* *Obtain Role-Based Accounts*
+* *Repurpose Deployed Dev Environment for Testing*
+
```

Comment:
Wasn't deploying the dev auth something we also considered?

Reply:
Yep! I included that as part of "repurpose Deployed Dev for testing". In the bullet point: "Can configure Dev to use dev auth, but keep impl a production-like environment". Did we consider doing it on impl, too?